### PR TITLE
EN-1590 add "migrate" method to storage; add "getCurrentSecretVersion…

### PR DESCRIPTION
**Pull request by igor-alekseev-dev**
…" method to InCrypt; fix some eslint warnings

Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [x] https://incountry.atlassian.net/browse/EN-1590

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [x] Please put the target release vehicle this needs to be bundled with.
- [x] Did you increment the version in package.json? What is the new version?

### CODE REVIEW

File: secret-key-accessor.js [hunk 1]
```diff
@@ -4,39 +4,54 @@ const { toPromise } = require('./utils');
...
- * Can return: 
+ * Can return:
  * - single key string
  * - KeyObject with diffrent verions of key
  * - Promise<string> or Promise<KeyObject> for any async jobs
- * 
+ *
  * @callback GetKeySecurelyCallback
  * @returns {string|SecretsObject|Promise<string>|Promise<SecretsObject>|unknown}
  */
```
Comment (RommeroInCountry) [inline secret-key-accessor.js]: please rename to match naming in constructor

File: secret-key-accessor.js [hunk 2]
```diff
@@ -54,12 +69,12 @@ class SecretKeyAccessor {
...
+    return this._getSecrets().then((so) => {
       const version = secretVersion !== undefined ? secretVersion : so.currentVersion;
-      const item = so.secrets.find(s => s.version == version);
-      return item !== undefined ? 
-        item : 
-        Promise.reject(new Error('Please provide secret key for this data'));
+      const item = so.secrets.find((s) => s.version === version);
+      return item !== undefined
+        ? item
+        : Promise.reject(new Error('Please provide secret key for this data'));
```
Comment (RommeroInCountry) [inline secret-key-accessor.js]: Can we please change the error to smth like `Secret not found for version <version>`? This way the error'd be more informative

File: storage.js [hunk 3]
```diff
@@ -230,12 +230,34 @@ class Storage {
     }
   }
 
+  /**
+   * @param {string} country - Country code.
+   * @param {number} limit - Find limit
+   * @returns {Promise<{ migrated: number, totalLeft: number }>}
+   */
+  async migrate(country, limit) {
```
Comment (vladimir-grezin) [inline storage.js]: Maybe we also need limit validation (ensure it's positive integer)?

Comment (igor-alekseev-dev) [reply storage.js]: I can add it to `find()` to cover more cases
@Rommero what do you think?

Comment (RommeroInCountry) [reply storage.js]: Let's add it to find for now

**Pull request merged**